content/applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst

```diff
@@ -1,101 +1,256 @@
-        and the number of periods you want to display at all times.
+To use |MPS| to manage the replenishment of a product, navigate to :menuselection:`Manufacturing app
+--> Planning --> Master Production Schedule`. At the top of the |MPS| page, click :guilabel:`Add a
+Product` to open the :guilabel:`Add a Product` pop-up window.
 
-Now, go to :menuselection:`Planning --> Master Production Schedule` and click on *add a
-product*. You can now define your safety stock target (= the stock you
-want to have on hand at the end of the period) and the minimum and
-maximum quantities that must or can be replenished in each period.
+.. important::
+   Products must be properly configured to be replenished through |MPS|.
 
-.. image:: use_mps/mps_1.png
-    :align: center
+   In the case of manufactured products, the :guilabel:`Manufacture` route must be selected in the
+   :guilabel:`Routes` section of the :guilabel:`Inventory` tab, on the product's page.
 
-In the MPS view, you can decide which information you would like to display
-by clicking on *rows*. For instance, the *Actual demand* will show you which
-quantity of products has already been ordered for the period, or *Available to Promise*,
-what can still be sold during that same period (what you plan to replenish - what
-is already sold during the period). You can also decide to hide rows if you like.
+   In the case of products that are purchased, the :guilabel:`Buy` route must be selected in the
+   :guilabel:`Routes` section of the :guilabel:`Inventory` tab, on the product's page. Additionally,
+   a vendor and the price they sell the product for must also be specified on the
+   :guilabel:`Purchase` tab.
 
-.. image:: use_mps/mps_2.png
-    :align: center
+On the pop-up window, select the product to add in the :guilabel:`Product` drop-down menu. If the
+product is replenished through manufacturing, select the product's |BoM| in the :guilabel:`Bill of
+Materials` field.
 
-Estimate your demand and launch replenishment
-=============================================
+.. note::
+   Selecting a BoM when adding a product to the |MPS| also adds any components listed on the BoM. If
+   it is not necessary to manage the replenishment of components through the |MPS|, simply leave the
+   :guilabel:`Bill of Materials` field blank.
 
-The next step is to estimate the demand for the chosen period. This is done
-in the *Forecasted Demand* Row. You can easily, at any time, compare the demand
-forecast with the actual demand (= confirmed sales). The demand forecast for a
-finished product will impact the indirect demand for its components.
+In the :guilabel:`Safety Stock Target` field, specify the minimum quantity of the product that
+should be kept available for orders at all times. For example, if there should always be 205 units of
+the product available for order fulfillment, enter `20` in the :guilabel:`Safety Stock Target`
```

Comment:
"If there should always be 205 units, enter `20` in Safety Stock"? This isn't really making sense to me, is this a typo?